ds3-sdk/src/main/java/com/spectralogic/ds3client/Ds3ClientBuilder.java

```diff
@@ -78,17 +78,17 @@ public static Ds3ClientBuilder create(final String endpoint, final Credentials c
      * @throws IllegalArgumentException
      */
     public static Ds3ClientBuilder fromEnv() throws IllegalArgumentException {
-        final String endpoint = System.getenv(ENDPOINT);
+        final String endpoint = System.getenv("ENDPOINT");
```

Comment:
Ahh, my bad.  I thought the environment variable was supposed to be set up as "ENDPOINT" but it's actually set up as "DS3_ENDPOINT".